Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -16,62 +16,164 @@
 use Wirecard\Oxid\Controller\Admin\Tab;
 use Wirecard\Oxid\Model\Transaction;
 use Wirecard\Oxid\Core\Helper;
+use Wirecard\Oxid\Core\TransactionHandler;
+use Wirecard\Oxid\Core\PaymentMethodFactory;
+use Wirecard\Oxid\Core\PaymentMethodHelper;
+use Wirecard\PaymentSdk\Transaction\SofortTransaction;
+use Wirecard\PaymentSdk\BackendService;
+use Wirecard\PaymentSdk\Config\Config;
 
 /**
  * Controls the view for the post-processing transaction tab.
  *
- * @since 1.0.0
+ * @since 1.0.1
  */
 class TransactionTabPostProcessing extends Tab
 {
+    const KEY_ACTION = 'action';
+    const KEY_AMOUNT = 'amount';
+
     /**
      * @var Transaction
      *
-     * @since 1.0.0
+     * @since 1.0.1
```

Comment:
this since tag should not change and stay 1.0.0 as well

Reply:
👍